### ISSUE TITLE
Build error in OSX Yosemite.

### ISSUE THREAD

**Issue by prashnts** [codyps/picocom#1]
Throws this error on `$ make NO_LDO=1` on Yosemite. What is missing? I cannot seem to find a library for this.

```
In file included from picocom.c:44:0:
term.h:103:40: fatal error: asm/termbits.h: No such file or directory
compilation terminated.
```

**Comment by codyps** [comment]
This is a fairly hacked up version of picocom that only supports some linux APIs. If you want to build on Mac OS X, you'll need to look at upstream (try the `trunk` branch).

**Comment by prashnts** [comment]
I did switch to `trunk`. Returns another build error. (Check screenshot.)
I tried to check `term.c` to find what was wrong. But I figure now, that baud rates 460800bps, and 921600bps are (maybe?) not supported in OSX? Because, removing the `DHIGH_BAUD` flag from MakeFile **did** lead to a successful build (yay).

I do not have much knowledge in this area, but, I'd like to learn, if I could fix this error, preferably by updating dependencies? 

PS: I needed this because I was installing `ino`, Arduino interface. It's working now.

![screen shot 2015-03-22 at 9 26 54 pm](https://cloud.githubusercontent.com/assets/5663391/6770061/32f03c96-d0da-11e4-9844-10c13dd95efd.png)
